schema/sqlite/v3/temporal/schema.sql

```diff
@@ -256,6 +257,15 @@ CREATE TABLE cluster_metadata (
 	PRIMARY KEY(metadata_partition)
 );
 
+CREATE TABLE cluster_metadata_info (
```

Comment:
do we not need version bump for sqlite?

Reply:
Do we official release the sqlite? I don't see versioned info. But I can add those

Reply:
good point. sqlite is not ready for use anyway. i see no need to bump version now.